src/components/AssetsTable/index.jsx

```diff
@@ -161,6 +161,7 @@ export default class AssetsTable extends React.Component {
             data-asset-id={asset.id}
             aria-label={displayText}
             onClick={this.onLockClick}
+            data-identifier="asset-lock-button"
```

Comment:
Is it possible for this to be lock/unlock depending on which class it is?

Reply:
Actually nvm, it will be possible to fetch lock/unlock with the regular CSS selector :) disregard.